src/api/remote.rs

```diff
@@ -214,6 +313,17 @@ impl TryFrom<String> for PollInterval {
     }
 }
 
+/// The integration's branch that you intend to scan
+#[derive(Debug, Clone, PartialEq, Eq, AsRef, Display, Deserialize, Serialize, new)]
+pub struct WatchedBranch(String);
+
+impl WatchedBranch {
+    /// The name of the watched branch
+    pub fn name(&self) -> String {
+        self.0.clone()
+    }
```

Comment:
It may be a better interface to just return the `&str` underlying `self.0` and then allow the caller to clone it if needed.

Reply:
This is correct, please make this change.
Reference: https://rust-lang.github.io/api-guidelines/flexibility.html?highlight=clone#caller-decides-where-to-copy-and-place-data-c-caller-control